thrift-confignode/src/main/thrift/confignode.thrift

```diff
@@ -21,6 +21,20 @@ include "rpc.thrift"
 namespace java org.apache.iotdb.confignode.rpc.thrift
 namespace py iotdb.thrift.confignode
 
+struct DataNodeRegisterReq {
+    1: required rpc.EndPoint endPoint
+}
+
+struct DataNodeRegisterResp {
+    1: required rpc.TSStatus registerResult
+    2: optional i32 dataNodeID
+}
+
+struct DataNodeInfo {
+  1: required i32 dataNodeID
+  2: required rpc.EndPoint endPoint
+}
```

Comment:
This struct may conflict with the `confignode/src/main/java/org/apache/iotdb/confignode/partition/DataNodeInfo.java,`
Actually, it confuses me.

Reply:
Renamed.